bodhi/server/templates/update.html

```diff
@@ -104,9 +104,7 @@
       $.each(data.data, function(i, result) {
 
         var name = result.testcase.name;
-        // we want to stash results by scenario when there is one, use name
-        // when there isn't
-        var scenario = result.testcase.name;
+        var scenario = 'no_scenario';
```

Comment:
Uhhhh...why this? Doesn't it mean that if we have multiple different tests, none of which includes a scenario key, we'll only display one of them (whichever we encountered last)?